src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleDataprocDAO.scala

```diff
@@ -92,7 +92,7 @@ class HttpGoogleDataprocDAO(appName: String,
     } {
       case e: HttpResponseException if e.getStatusCode == StatusCodes.NotFound.intValue => ()
       case e: GoogleJsonResponseException if e.getStatusCode == StatusCodes.BadRequest.intValue &&
-        e.getDetails.getMessage.contains("it has other pending delete operations against it") => ()
+        e.getDetails.getMessage.contains("pending delete") => ()
```

Comment:
still feels dirty :(

Reply:
It does, maybe we can add a call to check cluster status and delete only if not deleting. It's cleaner but would take longer.